sdk/com.ibm.sbt.web/src/main/webapp/js/sdk/_bridges/dojo-amd/RequestTransport.js

```diff
@@ -55,7 +55,19 @@ define([ "dojo/_base/declare", "dojo/_base/lang", "dojo/request", "../util" ], f
          * 
          */
         request : function(url,options) {
-            return request(url, options);
+        	var promise = new Promise();
+            promise.response = new Promise();
+            request(url, args).response.then(
+                function(response){
```

Comment:
The Dojo Request module returns a promise. This promise contains a property called response. The createResponse function in Transport.js, builds this property, so since the response property is returned automatically by the dojo request promise, there is no need to call createResponse.